spec/pubid_ieee/identifier_spec.rb

```diff
@@ -411,7 +413,7 @@ def context_description(ex)
 
     context "IEEE P802.11ajD8.0, August 2017" do
       let(:original) { "IEEE P802.11ajD8.0, August 2017" }
-      let(:pubid) { "IEEE P802.11aj/D8.0, August 2017" }
+      let(:pubid) { "IEEE Std P802.11aj/D8.0, August 2017" }
```

Comment:
The "Pxxxx" ones are not published standards -- they are preliminary projects, so they should not have "Std".